### ISSUE TITLE
Enigmatic Tree of Life spawning vegetation blocks that get destroyed when copy-pasting the structure

### ISSUE THREAD

**Issue by User-12691296** [EnigmaticaModpacks/Enigmatica6#4406]
### Modpack Version

1.0.0

### Describe your issue.

When I was experimenting in a creative world, I pasted the enigmatic tree of life multiple times, and each time, after the initial paste, the structure failed to form. Both of the times this happened, it was because of a limitation of MC's vegetation, where vegetation would die, or path blocks would be turned into dirt, or any of these reasons, the tree would fail to form.

1. The mushrooms. There are about 10 brown mushrooms throughout the tree, on the surface, and these are the most problematic. Nearly every time I pasted, some lag would intervene, and the mushroom block would finish its animation and place before the block underneath it. When this happened, the mushroom would break off into item form, and even this one mushroom not being there caused the whole structure to not form
2. The path blocks. Despite spawning this in the air, it sometimes intersected with trees, and when it did, the trees leaves would end up on top of the path blocks, trampling the path blocks, and causing the structure to not form

All in all, this bug isn't 100%, and it's preventable, but when I first pasted it, even in creative mode, I got really scared when it didn't form. I thought some update might have caused a desync, and that the pack was unfinishable. I can only imaging what it must feel like in survival mode, after many months of grinding expert, and all of your progress gets stopped by a mushroom. If not fixed, maybe JEI could show a warning, so players don't freak out and panic, thinking their pack is unfinishable.

### Have you modified the modpack?

No

### Did the issue happen in singleplayer or on a server?

Both

**Comment by MuteTiefling** [comment]
Hrm, I pasted it a few times and didn't have any issues. Guess I got lucky ;D

I'll take another look here and remove some of those blocks from the structure requirements. They'll still be part of the paste, just won't impact the tree.

**Comment by MuteTiefling** [comment]
ok, I pasted it again and had a nice lag spike that seemed to result in some shrooms popping off. 

I've removed them from the structure requirements. 

I can't replicate what you're talking about with the path blocks though. The only ones are inside the structure and there's absolutely nothing near them to make them revert.
![2022-03-14_09 10 23](https://user-images.githubusercontent.com/9543430/158178496-760809fc-380e-4c3e-9f52-7979f682394a.png)
